test/unit/org/apache/cassandra/streaming/StreamManagerTest.java

```diff
@@ -61,11 +61,11 @@ public static void setupClass()
     public void testUpdateStreamThroughput()
     {
         // Initialized value check
-        assertEquals(defaultStreamThroughputMebibytesPerSec * BYTES_PER_MEBIBYTE, StreamRateLimiter.getRateLimiterRateInBytes(), 0);
+        assertEquals(defaultStreamThroughputMebibytesPerSec * BYTES_PER_MEBIBYTE, StreamRateLimiter.getRateLimiterRateInBytes(), 0.01);
```

Comment:
nit: I believe these changes can be skipped and we can be consistent if we change in the test class the constant MEBIBYTES_PER_MEGABIT to (same as DataRateSpec)

private static final double MEBIBYTES_IN_BITS = Math.pow(2, 23);
    private static final double MEGABIT_IN_BITS = Math.pow(10, 6);
    private static final double MEBIBYTES_PER_MEGABIT = MEGABIT_IN_BITS / MEBIBYTES_IN_BITS;